src/commands/vault/add/processor.ts

```diff
@@ -59,7 +59,7 @@ export async function runAddTreasurer({
     guild_id: guildId,
     user_discord_id: user.id,
     vault_name: vaultName,
-    message: i.options.getString("message") ?? "",
+    message: i.options.getString("message") ?? "Add member as treasurer",
```

Comment:
dùng `||` để dù có emptry string nó cũng sẽ fallback qua vế bên phải, còn `??` chỉ có tác dụng với `null` và `undefined`
```suggestion
    message: (i.options.getString("message")?.trim()) || "Add member as treasurer",
```